.eslintrc.js

```diff
@@ -1,10 +1,11 @@
 module.exports = {
-  extends: '@mate-academy/eslint-config',
+  extends: "@mate-academy/eslint-config",
   env: {
-    jest: true
+    jest: true,
   },
   rules: {
-    'no-proto': 0
+    "no-proto": 0,
```

Comment:
The `no-proto` rule is set to 0, which disables the rule that prevents the use of `__proto__`. This can lead to potential security and performance issues. Consider enabling this rule unless you have a specific reason to allow `__proto__`.